syntaxes/aqua.tmLanguage.json

```diff
@@ -98,7 +98,12 @@
         {
           "//": "Defines tokens for service methods calls. Token is located between dot and open bracket",
           "name": "support.variable.method.aqua",
-          "match": "(?<=\\.)[a-z0-9_]+(?=\\()"
+          "match": "(?<=\\.)[a-z][a-zA-Z0-9_]+(?=\\()"
+        },
+        {
+          "//": "Defines tokens for constants used anywhere in code",
+          "name": "support.type.constant.aqua",
+          "match": "\\b[A-Z0-9_]+\\b"
```

Comment:
I think it should be like this: `"\\b^[A-Z][A-Z0-9_-]+\\b"`
Two changes:
1. Always start with a letter
2. Permit to have `-`

Reply:
It has to be star instead of plus then: `"\\b[A_Z][A-Z0-9_-]*\\b"`

Reply:
you're right!